.github/workflows/runtests.yml

```diff
@@ -11,36 +15,37 @@ jobs:
       matrix:
         os: [windows-latest]
         # os: [windows-latest, ubuntu-latest]
-        framework: [netcoreapp3.1, net6.0, net462]
+        framework: [net8.0, net6.0, netcoreapp3.1, net462]
     steps:
       - name: Checkout source
-        uses: actions/checkout@v2
```

Comment:
We could configure the dependabot to take care of these, similar [to this one](https://github.com/pytest-dev/pyfakefs/blob/main/.github/dependabot.yml).

Reply:
For actions we only need to specify the major version change, so I think the  dependabot might not be needed.

Reply:
Not needed, but helpful... otherwise you have to manually follow these version changes (this was just an observation, not related to this PR).

Reply:
Please, the idea is good and it is a useful service. 
Just evaluating how useful it will be in this case since all minor and patch/revision are automatically picked up.
Maybe a monthly or quarterly time span might be useful - will look into it when we add release workflow.